ios/use_react_native-0.60.rb

```diff
@@ -14,7 +14,8 @@ def include_react_native!(react_native, target_platform)
   pod 'React-DevSupport', :path => "#{react_native}/React"
 
   # fishhook was removed in 0.60.5
-  pod 'React-fishhook', :path => "#{react_native}/Libraries/fishhook" if File.exist?("#{react_native}/Libraries/fishhook")
+  fishhook = "#{react_native}/Libraries/fishhook"
+  pod 'React-fishhook', :path => fishhook if target_platform == :macos || File.exist?(fishhook)
```

Comment:
I've been meaning to ask you about this but it completely slipped my mind. The issue is that when you run `pod install` with `--project-directory=macos`, `cwd` is the path in which the command was invoked. Since `fishhook` gets resolved to `../node_modules/react-native/Libraries/fishhook`, `File.exist?(fishhook)` will always be `false` because it checks the parent folder of `cwd`. Is there a way to get the project directory so we can resolve to the correct path? Or do you think this is fine for now since we'll be moving to 0.61 and 0.62 soon?

Reply:
Hmm, that seems wrong. Perhaps that’s a CocoaPods bug that should be reported and for now we should jus `cd` into those dirs instead of use `--project-directory`?

Reply:
Never mind. I had a brain fart. Fixed now 😄